modules/rosa-policy-incident.adoc

```diff
@@ -1,17 +1,98 @@
 
 // Module included in the following assemblies:
 //
-// * rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc
-
+// * rosa_architecture/rosa_policy_service_definition/rosa-policy-shared-responsibility.adoc
 [id="rosa-policy-incident_{context}"]
 = Incident and operations management
 
+Red Hat is responsible for overseeing the service components required for default platform networking.
+AWS is responsible for protecting the hardware infrastructure that runs all of the services offered in the AWS Cloud. The customer is responsible for incident and operations management of customer application data and any custom networking the customer may have configured for the cluster network or virtual network.
+
+[cols= "2a,3a,3a",options="header"]
+|===
+
+|Resource
+|Service responsibilities
+|Customer responsibilities
+
+|Application networking
+|**Red Hat**
+
+- Monitor native OpenShift router
+service, and respond to alerts.
+|- Monitor health of application routes, and the endpoints behind them.
+- Report outages to Red Hat and AWS.
+
+|Virtual networking management
+|**Red Hat**
+
+- Monitor AWS load balancers, Amazon VPC subnets, and AWS service components necessary for default
+platform networking. Respond to alerts.
+|- Monitor health of AWS load balancer endpoints.
+- Monitor network traffic that is optionally configured through Amazon VPC-to-VPC connection, AWS VPN connection, or AWS
+Direct Connect for potential issues or
+security threats.
+
+|Virtual storage management
+|**Red Hat**
+
+- Monitor Amazon EBS volumes used for cluster nodes, and Amazon S3 buckets used for the ROSA service’s built-in container image
```

Comment:
I would say: "Monitor Amazon EBS volumes attached to cluster nodes" or something similar. One can interpret the current phrasing as indicating that "EBS volume = cluster node"